test/suites/dev/moonbase/test-xcm-v4/test-xcm-evm-with-dot.ts

```diff
@@ -0,0 +1,162 @@
+import "@moonbeam-network/api-augment";
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+
+import { alith } from "@moonwall/util";
+import { ApiPromise } from "@polkadot/api";
+import { u128 } from "@polkadot/types";
+import { PalletAssetsAssetAccount, PalletAssetsAssetDetails } from "@polkadot/types/lookup";
+import { hexToBigInt } from "@polkadot/util";
+import { Abi, encodeFunctionData } from "viem";
+import {
+  RELAY_SOURCE_LOCATION,
+  mockAssetBalance,
+  relayAssetMetadata,
+  verifyLatestBlockFees,
+} from "../../../../helpers/index.js";
+import {
+  RawXcmMessage,
+  XcmFragment,
+  XcmFragmentConfig,
+  descendOriginFromAddress20,
+  injectHrmpMessageAndSeal,
+  registerForeignAsset,
+} from "../../../../helpers/xcm.js";
+
+describeSuite({
+  id: "D014130",
+  title: "Mock XCM - Send EVM transaction through and pay with xcDOT",
+  foundationMethods: "dev",
+  testCases: ({ context, it, log }) => {
+    let sendingAddress: `0x${string}`;
+    let descendAddress: `0x${string}`;
+    let api: ApiPromise;
+    let assetId: u128;
+    let contractDeployed: `0x${string}`;
+    let contractABI: Abi;
+
+    const initialAlithBalance: bigint = 10_000_000_000_000_000n;
```

Comment:
This is not Alith balance, Alith in the scenario below is the sudo account.
```suggestion
    const initialSenderBalance: bigint = 10_000_000_000_000_000n;
```

Reply:
Wait, the test is mocking the balance for the descendAddress, which comes from  `descendOriginFromAddress20()`. As far as I understood, this will give us Alith's address. It is true that we pass `alith` as sudo account, so maybe the naming is confusing. I'll accept your suggestion but I'd like to clarify if calling `descendOriginFromAddress20()` with no parameters will give as Alith's address20.